### ISSUE TITLE
Solaris 11: test_ntlm fails

### ISSUE THREAD

**Issue by quanah** [heimdal/heimdal#378]
With Heimdal 7.5.0, patching in for 2b5b7cf, the NTLM test fails:

```
cat test_ntlm.log
test_ntlm: verify_ntlmv2
FAIL test_ntlm (exit status: 1)

cat test_ntlm.trs
:test-result: FAIL
:global-test-result: FAIL
:recheck: yes
:copy-in-global-log: yes

cat test-suite.log
============================================
   Heimdal 7.5.0: lib/ntlm/test-suite.log
============================================

# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: test_ntlm
===============

test_ntlm: verify_ntlmv2
FAIL test_ntlm (exit status: 1)
```

**Comment by quanah** [comment]
```
./test_ntlm --verbose
test_parse
test_keys
test_ntlm2_session_resp
test_targetinfo
test_ntlmv2
test_ntlm: verify_ntlmv2
quanah@sol11-build:~/git/sold/sold-master/SunOS5.11-i86pc/build/heimdal.i86pc_64/lib/ntlm$ echo $?
1
```

**Comment by lhoward** [comment]
This is a separate issue, fix coming up.

**Comment by lhoward** [comment]
Fixed in 590be3d7.
